Makefile.development

```diff
@@ -0,0 +1,49 @@
+# Default shell
+SHELL := /bin/bash
+
+# Conf vars
+GITHUB_API_KEY ?=
+WWW_PATH ?= /var/www
+LOGS ?= true
+
+#vars
+docker_container_name := doc-srcd-cont
+docker_image_name := quay.io/srcd/docs
+docker_shell := /bin/bash
+caddy_internal_port := 9090
+
+#tools
+docker_run := docker run --rm --detach
+docker_build := docker build -t
+docker_logs := docker logs --follow
+docker_stop := docker stop
+docker_tag := docker tag
+docker_access := docker exec -it
+git_last_commit := $(shell git show --format="%h" --no-patch)
+git_modified = $(shell git status --short)
+
+develop-run: build develop-stop
+	$(docker_build) $(docker_image_name) .;
+	$(docker_run) \
+		--name $(docker_container_name) \
+		--publish $(caddy_internal_port):$(caddy_internal_port) \
+		--env DEBUG_LOG=$(LOGS) \
+		--env GITHUB_API_KEY=$(GITHUB_API_KEY) \
+		$(docker_image_name);
+	$(docker_access) $(docker_container_name) $(docker_shell) || true;
+
```

Comment:
This appears to do two things that are different:

1. Run the container
2. Exec a shell into it

I'd separate it in two different actions or make sure the name covers both. Or create a new action composing run and exec a bash